CHANGELOG.md

```diff
@@ -78,6 +78,7 @@ New deprecation(s):
 
 ### Other
 
+- **General**: Expose prometheus metrics at ScaledJob like ScaledObject ([#4798](https://github.com/kedacore/keda/issues/4798))
```

Comment:
```suggestion
- **General**: Expose prometheus metrics for ScaledJob resources ([#4798](https://github.com/kedacore/keda/issues/4798))
```

Reply:
also, please move this to the `### New` section as it is a user facing change.

Reply:
Thanks!
Moved it.